app/build/app/js/app.js

```diff
@@ -70,6 +72,21 @@ angular.module('media_manager').controller('CollectionsController', [
         lc.deleteCollectionModal = CollectionBehavior.deleteCollectionModal;
         lc.actuallyDeleteCollection = CollectionBehavior.actuallyDeleteCollection;
         lc.isLoadingCollections = CourseCache.isLoadingCollections;
+
+        lc.dragControlListeners = {
+          orderChanged: function(event){
+
+            lc.collections.forEach(function(item, index, arr){
+              var newsort = index + 1;
+              if(item.sort_order !== newsort){
+                arr[index].sort_order = newsort;
+                Collection.update({id: item.id}, arr[index]);
```

Comment:
I could see this being done a few ways in the API, might be a good opportunity for us to do together on monday